open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/NewKeyMapFrame.java

```diff
@@ -0,0 +1,101 @@
+package io.opensphere.core.control.keybinding;
+
+import io.opensphere.core.hud.awt.AbstractInternalFrame;
+import javafx.application.Platform;
+import javafx.embed.swing.JFXPanel;
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.control.Tab;
+import javafx.scene.control.TabPane;
+import javafx.scene.control.TabPane.TabClosingPolicy;
+
+/**
+ * The Class KeyMapFrame. This class will show the control and shortcut keys.
+ */
+
+public class NewKeyMapFrame extends AbstractInternalFrame
+
+{
+    /** Serial. */
+    private static final long serialVersionUID = 1L;
+
+    /** The title of the window. */
+    public static final String TITLE = "Key Map";
+
+    /**
+     * 
+     * The Container panel. Since this JInternalFrame can be 'torn off' and uses
+     * the JInternalFrame's content pane, set the content pane to a JavaFX Panel
+     * through a JFX Dialog.
+     */
+
+    public NewKeyMapFrame()
+    {
+        super();
+        final JFXPanel fxPanel = new JFXPanel();
+
+        setSize(748, 610);
+        setPreferredSize(getSize());
+        setMinimumSize(getSize());
+        setTitle(TITLE);
+        setOpaque(false);
+
+        setIconifiable(false);
+        setClosable(true);
+        setResizable(false);
+        setDefaultCloseOperation(HIDE_ON_CLOSE);
+        setContentPane(fxPanel);
+        initAndShowGUI(fxPanel);
+    }
+
+    /**
+     * Takes the JAVA FX panel and puts it onto an FX thread for display.
+     * 
+     * @param fxPanel the panel to initialize.

```

Comment:
No need to give the name of the return variable

Reply:
removed